server/node.go

```diff
@@ -238,6 +241,10 @@ func (n *Node) start(rpcServer *rpc.Server, engines []engine.Engine,
 		log.Fatalf("unable to register node service with RPC server: %s", err)
 	}
 
+	// Start status monitor.
+	sub := n.ctx.EventFeed.Subscribe()
+	go n.status.StartMonitorFeed(sub)
```

Comment:
The goroutine is necessary because this is a consuming subscriber, consuming has to be asynchronous.  Would it make more sense to you if the goroutine was started inside the method itself?